### PR TITLE
refactor README.md

### PR DESCRIPTION
**Pull request by SamerKhshiboun**
Tracked by [LRS-676]

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,65 @@
...
+
+<hr>
+
+<div style="text-align:center">
+
+[![rolling][rolling-badge]][rolling]
+[![humble][humble-badge]][humble]
+[![foxy][foxy-badge]][foxy]
+[![galactic][galactic-badge]][galactic]
+[![eloquent][eloquent-badge]][eloquent]
```
Comment (Nir-Az) [inline README.md]: Lets remove dashing and eloquent,  also ubuntu 18

File: README.md [hunk 2]
```diff
@@ -1,15 +1,65 @@
...
+  * [ROS1 and ROS2 legacy](#legacy)
+  * [Installation](#installation)
+  * [Usage](#usage)
+     * [Starting the camera node](#start-camera-node)
+     * [Parameters](#parameters)
+     * [Topics](#topics)
+     * [Metadata Topic](#metadata)
+     * [Post-Processing Filters](#filters)
+     * [Available Services](#services)
+     * [Efficient intra-process communication](#intra-process)
```
Comment (Nir-Az) [inline README.md:39]: We can here add some coordination system section/point to doc.
Better to explain in the doc Vision sensors + IMU axis

File: README.md [hunk 3]
```diff
@@ -26,89 +76,174 @@ LibRealSense supported version: v2.51.1 (see [realsense2_camera release notes](h
...
+    Step 1: Install the ROS2 distribution 
+  </summary>
+  
+- #### Ubuntu 22.04:
+  - [ROS2 Humble](https://docs.ros.org/en/humble/Installation/Ubuntu-Install-Debians.html)
+- #### Ubuntu 20.04: 
+  - [ROS2 Foxy](https://docs.ros.org/en/foxy/Installation/Ubuntu-Install-Debians.html)
+  - [ROS2 Galactic](https://docs.ros.org/en/galactic/Installation/Ubuntu-Install-Debians.html)
+- #### Ubuntu 18.04 : 
+  - [ROS2 Dashing](https://docs.ros.org/en/dashing/Installation/Ubuntu-Install-Debians.html)
```
Comment (Nir-Az) [inline README.md]: We can drop this 2

File: README.md (hunk 1 above)
Comment (SamerKhshiboun) [reply README.md]: done

File: README.md (hunk 3 above)
Comment (SamerKhshiboun) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (SamerKhshiboun) [reply README.md:39]: done

File: README.md [hunk 4]
```diff
@@ -26,89 +76,277 @@ LibRealSense supported version: v2.51.1 (see [realsense2_camera release notes](h
...
+<details>
+  <summary>
+    Step 1: Install the ROS2 distribution 
+  </summary>
+  
+- #### Ubuntu 22.04:
+  - [ROS2 Humble](https://docs.ros.org/en/humble/Installation/Ubuntu-Install-Debians.html)
+- #### Ubuntu 20.04: 
+  - [ROS2 Foxy](https://docs.ros.org/en/foxy/Installation/Ubuntu-Install-Debians.html)
+  - [ROS2 Galactic](https://docs.ros.org/en/galactic/Installation/Ubuntu-Install-Debians.html)
```
Comment (Nir-Az) [inline README.md]: Let's add a `deprecated` notice on it (galactic)

File: README.md [hunk 5]
```diff
@@ -26,89 +76,277 @@ LibRealSense supported version: v2.51.1 (see [realsense2_camera release notes](h
...
+### Available Parameters:
+- For the entire list of parameters type `ros2 param list`.
+- For reading a parameter value use `ros2 param get <node> <parameter_name>` 
+  - For example: `ros2 param get /camera/camera depth_module.emitter_on_off`
+- For setting a new value for a parameter use `ros2 param set <node> <parameter_name> <value>`
+  - For example: `ros2 param set /camera/camera depth_module.emitter_on_off true`
 
-### Start the camera node
-To start the camera node in ROS:
+#### Parameters that can be modified during runtime:
```
Comment (Nir-Az) [inline README.md:209]: Maybe we can use the "collapse" method more?
this section takes many lines and nor sure everyone needs it.
We can ether collapse or point to another doc if it's very big.

Same for the new CS chapter.
What do you think?

Comment (SamerKhshiboun) [reply README.md:209]: we can use the collapse as much as we want.
The main reason I don't want to split it to another docs, is the keep it simple for searching.
Most of users search with CTRL+F, so it is easier to keep everything in one document.
What do you think ?

Comment (Nir-Az) [reply README.md:209]: Sure NP.
Does CTRL + F works inside "details"?

Comment (SamerKhshiboun) [reply README.md:209]: As I tested it now, NO :)

File: README.md (hunk 4 above)
Comment (SamerKhshiboun) [reply README.md]: done

File: README.md [hunk 6]
```diff
@@ -290,37 +493,18 @@ The launch file accepts a parameter, `intra_process_comms`, controlling whether
...
-**Other names and brands may be claimed as the property of others*
+</details>
+
+
+[rolling-badge]: https://img.shields.io/badge/-ROLLING-orange?style=flat-square&logo=ros
+[rolling]: https://docs.ros.org/en/rolling/Releases.html
+[humble-badge]: https://img.shields.io/badge/-HUMBLE-orange?style=flat-square&logo=ros
+[humble]: https://docs.ros.org/en/rolling/Releases/Release-Humble-Hawksbill.html
+[foxy-badge]: https://img.shields.io/badge/-FOXY-orange?style=flat-square&logo=ros
+[foxy]: https://docs.ros.org/en/rolling/Releases/Release-Foxy-Fitzroy.html
```
Comment (Nir-Az) [inline README.md]: Verify links

Comment (SamerKhshiboun) [reply README.md]: done

**Review by Nir-Az — APPROVED** [review]
Looks great!

**Pull request merged**